### PR TITLE
ScrollView experiments

### PR DESCRIPTION
**Pull request by siepra**
### Summary

I added an experiment for `ScrollView`.

I simply copied the example from official docs https://reactnative.dev/docs/scrollview and attached logger for every event prefixed with 'on' word.

I'm marking it as draft for now as I plan to continue with `FlatList` and `SectionList` within the same PR.

### CODE REVIEW

File: experiments-app/src/screens/ScrollViewEvents.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        style={styles.scrollView}
+        onMomentumScrollBegin={handleOnMomentumScrollBegin}
+        onMomentumScrollEnd={handleOnMomentumScrollEnd}
+        onScroll={handleOnScroll}
+        onScrollBeginDrag={handleOnScrollBeginDrag}
+        onScrollEndDrag={handleOnScrollEndDrag}
+        onScrollToTop={handleOnScrollToTop}
+      >
+        <Text style={styles.text}>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
```
Comment (mdjastrzebski) [inline experiments-app/src/screens/ScrollViewEvents.tsx:31]: Is that text long enough to make the screen larger than viewport? For this experiment to work well the content should take 2+ viewports.

File: experiments-app/src/screens/ScrollViewEvents.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+const handleOnMomentumScrollBegin = buildEventLogger('onMomentumScrollBegin');
+const handleOnMomentumScrollEnd = buildEventLogger('onMomentumScrollEnd');
+const handleOnScroll = buildEventLogger('onScroll');
+const handleOnScrollBeginDrag = buildEventLogger('onScrollBeginDrag');
+const handleOnScrollEndDrag = buildEventLogger('onScrollEndDrag');
+const handleOnScrollToTop = buildEventLogger('onScrollToTop');
+
+export function ScrollViewEvents() {
+  return (
+    <SafeAreaView style={styles.container}>
```
Comment (mdjastrzebski) [inline experiments-app/src/screens/ScrollViewEvents.tsx:20]: Could you also add similar experiment for flat list. We would want to support both of these.

File: experiments-app/src/screens/ScrollViewEvents.tsx (hunk 1 above)
Comment (siepra) [reply experiments-app/src/screens/ScrollViewEvents.tsx:31]: It is twice the viewport on my Pixel 6 device. I can make this text longer though.

**Comment by siepra** [comment]
Oh, sorry for not following the commit naming convention. I focused on the branch name instead (my bad). I promise to obey from now on

**Review by mdjastrzebski — APPROVED** [review]
Looks good! I've done some minor tweaks to matching code style of the other experiments.

**Pull request merged**